packages/block-editor/src/store/actions.js

```diff
@@ -232,17 +232,23 @@ export function selectBlock( clientId, initialPosition = 0 ) {
 
 /**
  * Yields action objects used in signalling that the block preceding the given
- * clientId should be selected.
+ * clientId or it's first parent from bottom to top should be selected.
  *
- * @param {string} clientId Block client ID.
+ * @param {string}  clientId      Block client ID.
+ * @param {boolean} orFirstParent If true, select the first parent if there is no previous block.
  */
 export const selectPreviousBlock =
-	( clientId ) =>
+	( clientId, orFirstParent = false ) =>
 	( { select, dispatch } ) => {
 		const previousBlockClientId =
 			select.getPreviousBlockClientId( clientId );
 		if ( previousBlockClientId ) {
 			dispatch.selectBlock( previousBlockClientId, -1 );
+		} else if ( orFirstParent ) {
```

Comment:
Why make this an option?

Reply:
Since this method is named `getPreviousBlock` the default behavior should remain just that, and going up the tree to select the parent is an option that we use when we _desperately need to focus something_. Otherwise we should get nothing if there is no previous block.